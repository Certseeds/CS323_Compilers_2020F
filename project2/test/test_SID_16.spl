int main(){
int a,b;
    while(0) {
        if(a) {
            int c = 1;
        } else if (b) {
            int d = a;
        }
        else {
            float c = 1.1;
        }
    }
    return 0;
}