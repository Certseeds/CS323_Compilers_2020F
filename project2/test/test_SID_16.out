Error type 3 at Line 10: redefine variable: c
